plugins/processors/awsappsignals/internal/attributes/attributes.go

```diff
@@ -5,38 +5,20 @@ package attributes
 
 const (
 	// aws attributes
-	AWSLocalService        = "aws.local.service"
-	AWSLocalOperation      = "aws.local.operation"
-	AWSRemoteService       = "aws.remote.service"
-	AWSRemoteOperation     = "aws.remote.operation"
-	AWSRemoteTarget        = "aws.remote.target"
-	AWSHostedInEnvironment = "aws.hostedin.environment"
+	AWSLocalService             = "aws.local.service"
+	AWSLocalOperation           = "aws.local.operation"
+	AWSRemoteService            = "aws.remote.service"
+	AWSRemoteOperation          = "aws.remote.operation"
+	AWSRemoteTarget             = "aws.remote.target"
+	AWSRemoteResourceIdentifier = "aws.remote.resource.identifier"
+	AWSRemoteResourceType       = "aws.remote.resource.type"
+	AWSHostedInEnvironment      = "aws.hostedin.environment"
 
 	// resource detection processor attributes
 	ResourceDetectionHostId   = "host.id"
 	ResourceDetectionHostName = "host.name"
 	ResourceDetectionASG      = "ec2.tag.aws:autoscaling:groupName"
 
-	// kubernetes resource attributes
-	K8SDeploymentName  = "k8s.deployment.name"
-	K8SStatefulSetName = "k8s.statefulset.name"
-	K8SDaemonSetName   = "k8s.daemonset.name"
-	K8SJobName         = "k8s.job.name"
-	K8SCronJobName     = "k8s.cronjob.name"
-	K8SPodName         = "k8s.pod.name"
-	K8SRemoteNamespace = "K8s.RemoteNamespace"
-
-	// ec2 resource attributes
-	EC2AutoScalingGroupName = "EC2.AutoScalingGroupName"
-	EC2InstanceId           = "EC2.InstanceId"
-
-	// hosted in attribute names
-	HostedInClusterNameEKS = "HostedIn.EKS.Cluster"
-	HostedInClusterNameK8s = "HostedIn.K8s.Cluster"
-	HostedInK8SNamespace   = "HostedIn.K8s.Namespace"
-	HostedInEC2Environment = "HostedIn.EC2.Environment"
-	HostedInEnvironment    = "HostedIn.Environment"
-
-	// sdk attributes
-	MetricAttributeSDKMetadata = "SDK"
+	AWSLocalEnvironment  = "aws.local.environment"
```

Comment:
nit: move these to the aws attributes section and put them together with the other `aws.local` and `aws.remote` attribute consts

Reply:
Done.